databricks.json

```diff
@@ -0,0 +1,3602 @@
+                    "example_values": [
+                        1024
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.catalog",
+                    "data_type": "string",
+                    "example_values": [
+                        "samples"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.schema",
+                    "data_type": "string",
+                    "example_values": [
+                        "nyctaxi"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.disposition",
+                    "data_type": "string",
+                    "example_values": [
+                        "INLINE"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.result.data_array.*",
+                    "data_type": "string",
+                    "example_values": [
+                        "10282"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.manifest.truncated",
+                    "data_type": "numeric",
+                    "example_values": [
+                        false
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        2
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        2
+                    ]
+                }
+            ],
+            "render": {
+                "type": "custom",
+                "width": 10,
+                "height": 5,
+                "view": "databricks_view.display_query_results",
+                "title": "Perform Query"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "execute notebook",
+            "description": "Execute a Databricks notebook",
+            "type": "generic",
+            "identifier": "execute_notebook",
+            "read_only": false,
+            "parameters": {
+                "notebook_path": {
+                    "description": "The path of the notebook to be run in the Databricks workspace or remote repository",
+                    "data_type": "string",
+                    "required": true,
+                    "order": 0
+                },
+                "git_source": {
+                    "description": "An optional specification for a remote repository containing the notebook",
+                    "data_type": "string",
+                    "order": 1
+                },
+                "existing_cluster_id": {
```

Comment:
Some of these parameters explicitly say they're optional (git_source and libraries for example) but a lot more of them are actually not saying required=true also. Would be good to be consistent

Reply:
Fixing